README.md

```diff
@@ -15,7 +15,7 @@
 <a href="https://www.linkedin.com/company/quesma">LinkedIn</a> | 
 <a href="https://twitter.com/QuesmaOrg">Twitter</a> | 
 <a href="https://www.youtube.com/@QuesmaOrg">YouTube</a> | 
-<a href="https://eap.quesma.com/eap-docs">Docs</a>
+<a href="https://docs.quesma.com/eap-docs">Docs</a>
```

Comment:
Should not we change this `eap-docs` to docs as well? (I know this also requires link fix)

Reply:
I'll do it in a follow-up PR